packages/lodestar/src/network/reqresp/reqResp.ts

```diff
@@ -59,8 +59,9 @@ export class ReqResp extends (EventEmitter as IReqEventEmitterClass) implements
       for (const encoding of Object.values(ReqRespEncoding)) {
         this.libp2p.handle(createRpcProtocol(method, encoding), async ({connection, stream}) => {
           const peerId = connection.remotePeer;
+          // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
With new eslint, you can just add "void" in front of promise you don't wan't to await